### PR TITLE
[PDMO-94]-Show deleted observations at the bottom.

### PR DESCRIPTION
**Pull request by Farhanisto**
## JIRA TICKET NAME:

[PDMO-94 Show deleted observations at the bottom](https://issues.openmrs.org/browse/PDMO-94)

## SUMMARY:
Deleted observations should be at the bottom of the list

### CODE REVIEW

File: app/css/obsadmin.css [hunk 1]
```diff
@@ -849,6 +859,13 @@ input[type="radio"]{
 }
 .deletedObs{
   text-decoration: line-through;
+  position: absolute;
+  bottom: 0;
+  
+}
```
Comment (Charpell) [inline app/css/obsadmin.css]: Fix double newline

**Comment by Farhanisto** [comment]
Thanks @Charpell . I have corrected

File: app/js/components/encounters/observations.jsx [hunk 2]
```diff
@@ -164,16 +164,16 @@ export default class Observations extends React.Component {
                             {
                               groupMembersArray.map(observation => {
                                 return (
-                                  <tr className={(observation.voided) ? 'deletedObs' : ''} key={observation.uuid}>
-                                    <a>
+                                  <tr className={(observation.voided) ? 'deletedObs' : 'undeletedobs'} key={observation.uuid}>
+
```
Comment (patlub) [inline app/js/components/encounters/observations.jsx]: As @Charpell pointed out. More unnecessary blank lines here.

**Review by patlub — CHANGES_REQUESTED** [review]

**Comment by Farhanisto** [comment]
Thanks @patlub . fixed

**Comment by joel-ace** [comment]
Looks Good

**Comment by Farhanisto** [comment]
@dkayiwa kindly look into this

File: app/js/components/encounters/observations.jsx [hunk 3]
```diff
@@ -100,7 +100,7 @@ export default class Observations extends React.Component {
               {
                 filteredObs && filteredObs.length > 0 &&
                 filteredObs.map((observation) => (
-                  <tr className={(observation.voided) ? 'deletedObs' : ''}>
+                  <tr className={(observation.voided) ? 'deletedObs' : 'undeletedobs'}>
```
Comment (dkayiwa) [inline app/js/components/encounters/observations.jsx]: undeletedobs is better named as undeletedObs

**Comment by dkayiwa** [comment]
@Farhanisto this pull request looks like it contains more than "Show deleted observations at the bottom".

File: app/js/components/encounters/observations.jsx (hunk 3 above)
Comment (Farhanisto) [reply app/js/components/encounters/observations.jsx]: making changes

**Comment by Farhanisto** [comment]
@dkayiwa this happened after I squeshed 2 commit messages that I got after making the pull request

File: app/js/components/encounters/observations.jsx (hunk 3 above)
Comment (dkayiwa) [reply app/js/components/encounters/observations.jsx]: Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

**Comment by dkayiwa** [comment]
@Farhanisto so can you fix it? The pull request should have only the changes required by the ticket. If you need to make more unrelated changes, you can always create a new ticket for them.

**Comment by Farhanisto** [comment]
alright thanks.

**Comment by Farhanisto** [comment]
@dkayiwa I have sorted this out. Kindly check

**Pull request merged**